### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by bemailloux**
Unit tests only. (no visual diff)

### CODE REVIEW

File: pagination.js [hunk 1]
```diff
@@ -58,6 +58,7 @@ class Pagination extends RtlMixin(Localizer(LitElement)) {
 		this.maxPageNumber = 1;
 	}
 
+	// TODO: page number types are inconsistent depending on if event is fired from arrows or from Enter keypress
```
Comment (bemailloux) [inline pagination.js:61]: Will tackle this in a future PR

File: test/pagination.test.js [hunk 2]
```diff
@@ -32,4 +32,272 @@ describe('pagination', () => {
...
+		it('should render page number and max page number correctly', async() => {
+			const el = await fixture(html`<d2l-labs-pagination pageNumber="3" maxPageNumber="8"></d2l-labs-pagination>`);
+			const pageInput = el.shadowRoot.querySelector('d2l-input-text');
+			const maxPageIndicator = el.shadowRoot.querySelector('span.page-max');
+			expect(pageInput.value).to.equal('3');
+			expect(maxPageIndicator.innerText).to.equal('∕ 8'); // this isn't a regular slash
+
+			// by default, it should not render the page size selector
+			const pageSizeSelector = el.shadowRoot.querySelector('select');
+			expect(pageSizeSelector).to.be.null;
```
Comment (anhill-D2L) [inline test/pagination.test.js:46]: Why is this null instead of whatever is set to be the default page size?

Comment (bemailloux) [reply test/pagination.test.js:46]: `showItemCountSelect` was not set, so it should not show the page size selector at all

File: test/pagination.test.js [hunk 3]
```diff
@@ -32,4 +32,272 @@ describe('pagination', () => {
...
+					expect(rightButton.disabled).to.be.true;
+				});
+
+				it('should disable both buttons if there are 0 pages', async() => {
+					const {leftButton, rightButton} = await getPaginationEl(0, 0, 'ltr');
+					expect(leftButton.disabled).to.be.true;
+					expect(rightButton.disabled).to.be.true;
+				});
+
+				it('should disable neither button if current page is not first or last', async() => {
```
Comment (anhill-D2L) [inline test/pagination.test.js:117]: This does a good job explaining under what conditions youd want to disable the buttons

File: test/pagination.test.js (hunk 2 above)
Comment (anhill-D2L) [reply test/pagination.test.js:46]: Cool, I didn't know you could have pagination without being able to set the page size

File: test/pagination.test.js [hunk 4]
```diff
@@ -32,4 +32,272 @@ describe('pagination', () => {
 			runConstructor('d2l-labs-pagination');
 		});
 	});
+
+	describe('render', () => {
+		it('should render page number and max page number correctly', async() => {
+			const el = await fixture(html`<d2l-labs-pagination pageNumber="3" maxPageNumber="8"></d2l-labs-pagination>`);
```
Comment (anhill-D2L) [inline test/pagination.test.js:38]: Nitpick: the html looks more readable when it's on separate lines like in the test below

Comment (bemailloux) [reply test/pagination.test.js:38]: Will fix in next PR

File: test/pagination.test.js [hunk 5]
```diff
@@ -32,4 +32,272 @@ describe('pagination', () => {
...
+					const {leftButton, rightButton} = await getPaginationEl(2, 3, 'ltr');
+					expect(leftButton.disabled).to.be.false;
+					expect(rightButton.disabled).to.be.false;
+				});
+			});
+
+			describe('rtl', () => {
+				// in rtl, the "left" button is actually on the right side and mirrored so it looks like a right arrow
+				// so the "left" button is still the one that is disabled, even though it's really the right button
+				// (and vice-versa for the "right" button, which is actually on the left)
```
Comment (BhamelinD2L) [inline test/pagination.test.js:127]: I appreciate the explanation, makes it much easier to understand what's going on here

Comment (bemailloux) [reply test/pagination.test.js:127]: I was confused at first too, which is why I added it :)

**Pull request merged**